### ISSUE TITLE
Layout/ClosingParenthesisIndentation error with newlines on 0.58.2

### ISSUE THREAD

**Issue by jaysonesmith** [rubocop-hq/rubocop#6127]
## Information

When refactoring some code, Rubocop threw an error for the Layout/ClosingParenthesisIndentation cop.

Offending code is: 

```ruby
  def foo
    JSON.parse(
      
    )
  end
```

Error is not thrown if parens are on the same line.

```ruby
  def foo
    JSON.parse()
  end
```

## Expected behavior

Standard Rubocop alerting without error.

## Actual behavior

Error:

> An error occurred while Layout/ClosingParenthesisIndentation cop was inspecting /Users/jsmith/code/go/src/github.com/brightcove/bolt-playback-automation/lib/bolt_video.rb:10:4. To see the complete backtrace run rubocop -d. 1 error occurred: An error occurred while Layout/ClosingParenthesisIndentation cop was inspecting /Users/jsmith/code/go/src/github.com/brightcove/bolt-playback-automation/lib/bolt_video.rb:10:4. Errors are usually caused by RuboCop bugs. Please, report your problems to RuboCop's issue tracker. https://github.com/rubocop-hq/rubocop/issues Mention the following information in the issue report: 0.58.2 (using Parser 2.5.1.2, running on ruby 2.4.1 x86_64-darwin16)

## Steps to reproduce the problem

Run Rubocop against:

```ruby
  def foo
    JSON.parse(
      
    )
  end
```

## RuboCop version

0.58.2 (using Parser 2.5.1.2, running on ruby 2.4.1 x86_64-darwin16)

**Comment by jaysonesmith** [comment]
Thanks @tatsuyafw!!
